src/client/components/HealthCareApp.jsx

```diff
@@ -205,14 +205,19 @@ class HealthCareApp extends React.Component {
       buttons = (
         <div className="row progress-buttons">
           <div className="small-6 medium-5 columns">
-            {nextButton}
+            <ProgressButton
+                onButtonClick={this.handleContinue}
+                buttonText="Get Started"
+                buttonClass="usa-button-primary"
+                afterText="»"/>
           </div>
         </div>
       );
     } else if (this.props.location.pathname === '/submit-message') {
       buttons = (
         <div className="row progress-buttons">
           <div className="small-6 medium-5 columns">
+            {/* TODO: Figure out where this button should take the user. */}
```

Comment:
Maybe back to vets.gov for now since vets.gov/healthcare doesn't have anything and vets.gov/healthcare/apply/ is just the landing page for the application?